src/Event.php

```diff
@@ -173,7 +172,7 @@ public function __construct()
      */
     public function getId(): string
     {
-        return $this->id;
+        return (string) $this->id;
     }
```

Comment:
Maybe we could deprecate this method and add a `getEventId`?

Reply:
I though about this, but I didn't like it because what would happen is `$event->getEventId()` which is redundant. Instead, I propos to deprecate the returning of a `string` from this method like we did for `Options::getDsn()` e.g.:

```php
if ($returnAsString) {
    return (string) $this->id;
} else {
    return $this->id;
}
```

Reply:
That's doable, but only if we suppress all deprecations around in our code too.

Reply:
Obviously 👍 Done in the latest commit